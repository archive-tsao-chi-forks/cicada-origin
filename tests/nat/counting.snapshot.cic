@def n zero
@def one add1(n)

@show one

@show @the Nat add1(add1(zero))
