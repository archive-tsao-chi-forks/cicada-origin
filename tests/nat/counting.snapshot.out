the Nat 1
the Nat 2

